src/types/models.ts

```diff
@@ -354,3 +354,14 @@ export type NoiseThresholds = {
    */
   noise_threshold_nrs?: number
 }
+
+export interface DeviceModel {
```

Comment:
Wondering if this may be better to go in `seamapi-types` so we don't have type defs spread across projects. Minor though as this is technically an unstable api still (I think; CMIIW)

Reply:
Mostly following the existing trend here with the other models. Really, I wish there was a way we could generate these types from the openapi spec instead of having to define them here.